app/Console/Kernel.php

```diff
@@ -19,6 +19,7 @@ class Kernel extends ConsoleKernel
         'App\Console\Commands\ImportCSV',
         'App\Console\Commands\ImportVCards',
         'App\Console\Commands\LangGenerate',
+        'App\Console\Commands\MoveAvatars',
```

Comment:
Perhaps we could put the commands that should only be done once if a proper folder, like `Commands\OneTime` or something.